django/db/backends/mysql/schema.py

```diff
@@ -80,10 +80,20 @@ def _delete_composed_index(self, model, fields, *args):
                 )
         return super(DatabaseSchemaEditor, self)._delete_composed_index(model, fields, *args)
 
-    def _alter_column_type_sql(self, table, old_field, new_field, new_type):
-        # Keep null property of old field, if it has changed, it will be handled separately
-        if old_field.null:
+    def _set_field_new_type_null_status(self, field, new_type):
+        """
+        Keep null property of old field, if it has changed, it will be handled separately
```

Comment:
Keep the null property of the old field. If it has changed, .... separately.
(also please wrap at 79 chars)